dev/com.ibm.websphere.appserver.thirdparty.eclipselink.3.0/resources/org/eclipse/persistence/exceptions/i18n/ConversionExceptionResource.nlsprops

```diff
@@ -0,0 +1,47 @@
+###############################################################################
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+# # {0} description of each insert field
+# MSG_DESCRIPTIVE_NAME_CWSJX0000=CWSJX0000I: This is a message with inserts {0}
+# MSG_DESCRIPTIVE_NAME_CWSJX0000.explanation=Explanation text for the message
+# MSG_DESCRIPTIVE_NAME_CWSJX0000.useraction=User action text for the message
+#
+#CMVCPATHNAME org/eclipse/persistence/exceptions/i18n/ConversionExceptionResource.nlsprops
+#COMPONENTPREFIX None
+#COMPONENTNAMEFOR None - generated from open source project
+#ISMESSAGEFILE TRUE
+#NLS_ENCODING=UNICODE
+#
+# NLS_MESSAGEFORMAT_ALL
+#
+#   Strings in this file which contain replacement variables are processed by the MessageFormat
+#   class (single quote must be coded as 2 consecutive single quotes ''). Strings in this file 
+#   which do NOT contain replacement variables are NOT processed by the MessageFormat class 
+#   (single quote must be coded as one single quote '). 
+#
+# -------------------------------------------------------------------------------------------------
+3001=The object [{0}], of class [{1}], could not be converted to [{2}].
+
+3002=The object [{0}], of class [{1}], from mapping [{2}] with descriptor [{3}], could not be converted to [{4}].
+
+3003=Incorrect date format: [{0}] (expected [YYYY-MM-DD])
+
+3004=Incorrect time format: [{0}] (expected [HH:MM:SS])
+
+3005=Incorrect timestamp format: [{0}] (expected [YYYY-MM-DD HH:MM:SS.NNNNNNNNN])
+
+3006=[{0}] must be of even length to be converted to a byte array.
+
+3007=The object [{0}], of class [{1}], could not be converted to [{2}].  Ensure that the class [{2}] is on the CLASSPATH.  You may need to use alternate API passing in the appropriate class loader as required, or setting it on the default ConversionManager
```

Comment:
Global comments:
- Use "cannot" instead of "could not"
- Use "might" instead of "may" unless denoting permission
- If possible, use the format "the _variable_ noun" 

Also, add a period to the end and clarify the use of "setting" ("set"?). Suggested rewrite: 
The [{0}] object of the [{1}] class cannot be converted to [{2}].  Ensure that the [{2}] class is on the CLASSPATH.  You might need to use alternate API passing in the appropriate class loader as required, or set it on the default ConversionManager.

Reply:
Done